integrationExamples/gpt/userId_example.html

```diff
@@ -134,6 +134,31 @@
                 // },
                 userSync: {
                     userIds: [{
+                        name: "pubProvided",
```

Comment:
can we chance "pubProvided" to "pubProvidedId", that matches the convention the other id vendors are using.